cardano-cli/src/Cardano/CLI/EraBased/Options/Genesis.hs

```diff
@@ -291,6 +301,14 @@ pGenesisCreateTestNetData envCli =
                              ]
         , Opt.value 500_000_000_000
         ]
+    pRelays :: Parser FilePath
+    pRelays =
+      Opt.strOption $ mconcat
+        [ Opt.long "relays"
```

Comment:
I chose a shorter flag name than `create-staked`'s `relay-specification-file`. I feel the longer name didn't bring much value. We already chose to use different names in `create-testnet-data` before, so this should not be a surprise to users.

Reply:
Why two different names for the same file?

Reply:
Not sure what you mean @carbolymer: there is a single name in `create-testnet-data` and one single name in `create-staked`. I simply chose a shorter, I think better, name for `create-testnet-data`.